source/slides/16_monitoring.rst

```diff
@@ -0,0 +1,92 @@
+.. _16_monitoring:
+
+Monitoring
+==========
+
+Definitions
+-----------
+
+* Specificity
+* Sensitivity
+* Expected Value
+
+Specificity
+-----------
+
+* Complement to the False-Negative Rate
+* Measures true negatives against identified negatives
+* Often called the True-Negative Rate
+
+Sensitivity
+-----------
+
+* Complement to the False-Positive Rate
+* Measures true positives against identified positives
+* Often called the True-Positive Rate
+
+Car Alarms
+----------
+
+* Car alarms have a low sensitivity rate (They don't go off when they should)
+* They also have a low specificity rate (They go off when they shouldn't)
+
+Smoke Alarms
+------------
+
+* Smoke alarms have a high sensitivity rate (they go off when there is smoke)
+* Smoke alarms have a high specificity rate (there is usually smoke when
+  they go off)
+
+Expected Value
+--------------
+
+Let **X** be a random variable, then **E(X)** is the mean of X.
+**E(i)** for some **i** in **X** is the probability of **i** occurring
+multiplied by the value of **i**
+
+Suppose **X** is the results of a fair 6-sided die, then **E(X)** is
+**3.5**. **E(1) = 1/6**, **E(2) = 2/6 = 1/3**, and so on.
+
+Note that **E(X) = sum(E(i) for all in in X)**, or
+
+**1/6 + 2/6 + 3/6 + 4/6 + 5/6 + 6/6 = 21/6 = 3 and 3/6 = 3.5**
+
+
+Example
+-------
+
+Suppose you have a server you know gets a lot of network connections.
+
+Thinking ahead, you bumped **nf_conntrack_max**, and have a nagios check
+that determines if you can open a new connection. Additionally, you
```

Comment:
You might break this paragraph up a little to make it easier to read on the slide.